docs/developer-walk-through.rst

```diff
@@ -1,140 +1,14 @@
-Developer Walk-through
-======================
+Other stuff
+===========
 
 .. toctree::
    :maxdepth: 2
    :caption: Contents
 
-   main-developer-workflow
-   how-to-add-a-new-script-to-the-cea
```

Comment:
I think this chapter is pretty important for new developers (or as a reference for older developers) - where is it added again?
(speakig of `how-to-add-a-new-script-to-the-cea`)

maybe we can add these in a sort of knowledge-base style catalog?

Reply:
yes it is kept already but re-organized